config/helics-config.h.in

```diff
@@ -36,5 +36,5 @@ SPDX-License-Identifier: BSD-3-Clause
 #define HELICS_VERSION_STRING "${HELICS_VERSION_STRING}"
 #define HELICS_DATE "${HELICS_DATE}"
 #define HELICS_COMPILER_VERSION "${HELICS_COMPILER_VERSION}"
-#define HELICS_BUILD_FLAGS "${HELICS_BUILD_FLAGS_OUTPUT}"
+#define HELICS_BUILD_FLAGS "${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_RELEASE} ${CMAKE_STATIC_LINKER_FLAGS} ${EXTRA_BUILD_FLAGS}"
```

Comment:
Are release builds that only ones that we really care about getting all of the flags for?

Reply:
I made it be a compile time switch based on NDEBUG